.travis.yml

```diff
@@ -0,0 +1,74 @@
+language: cpp
+dist: xenial
+git:
+  depth: 5
+  quiet: true
+
+python: 3.7
+
+
+
+matrix:
+  include:
+  # todo
+    # - name: Static checks (clang-format)
+    #   env: STATIC_CHECKS=yes
+    #   os: linux
+    #   compiler: gcc
+    #   addons:
+    #     apt:
+    #       sources:
+    #         - llvm-toolchain-xenial-8
+    #       packages:
+    #         - clang-format-8
+    #   script:
+    #    - $CXX --version
+    #    - mkdir build && cd build
+    #    - cmake -DBUILD_TESTING=ON ..
+    #    - make
+
+    # todo
+    # - name: osx
+    #   os: osx
+    #   osx_image: xcode11
+    #   compiler: clang
+    #   script:
+    #    - mkdir build && cd build
+    #    - cmake  ..
+    #    - make
+
+    - name: linux (clang)
+      os: linux
+      compiler: clang++
+      script:
+       - mkdir build && cd build
+       - cmake  ..
+       - make
```

Comment:
Travis supplies 2 cores to each VM so you can run this with make -j $(nproc) to get a little bit more speed